src/Access/Common/AccessType.h

```diff
@@ -142,6 +142,7 @@ enum class AccessType
     M(ACCESS_MANAGEMENT, "", GROUP, ALL) \
     M(SHOW_NAMED_COLLECTIONS, "SHOW NAMED COLLECTIONS", NAMED_COLLECTION, NAMED_COLLECTION_CONTROL) \
     M(SHOW_NAMED_COLLECTIONS_SECRETS, "SHOW NAMED COLLECTIONS SECRETS", NAMED_COLLECTION, NAMED_COLLECTION_CONTROL) \
+    M(USE_NAMED_COLLECTION, "USE NAMED COLLECTION", NAMED_COLLECTION, NAMED_COLLECTION_CONTROL) \
```

Comment:
Can't we shorten this name to just `NAMED_COLLECTION`? Because usually we don't prefix the names of privileges with the word `USE`.

Reply:
If someone confuses them and grant `NAMED_COLLECTION` instead if `NAMED_COLLECTION_CONTROL` they will get lesser access rights, so that doesn't seem too dangerous. And we have a lot of other access types like `FILE` or `CLUSTER` where we don't write `USE` or `USAGE` in their names.

Also `GRANT USE NAMED COLLECTION` looks like we have a command `USE NAMED COLLECTION`.

Reply:
Ok, agree.

Reply:
Maybe it's worth to introduce something like `NAMED_COLLECTION_USAGE` and make it an alias of `NAMED_COLLECTION`.

Reply:
And one more thing about the privilege `NAMED COLLECTION CONTROL` - it's dangerous but it doesn't look so. Probably it's better to rename it to `NAMED COLLECTION ADMIN` because the word `ADMIN` sounds more scary so people will hardly write it by mistake.